india_compliance/gst_india/utils/test_gstin_info.py

```diff
@@ -0,0 +1,123 @@
+import unittest
+from unittest.mock import Mock, patch
```

Comment:
You might wanna try https://github.com/getsentry/responses for better http response mocking and request validating. 😄